### PR TITLE
Support for rails 4 

### PR DESCRIPTION
**Pull request by Intrepidd**
In response to issue #8

Seems to work out of the box with only this gemspec change.

### CODE REVIEW

**Comment by linjunpop** [comment]
@Intrepidd, thank you. :beers: 

Could you do `s.add_dependency "railties", ">= 3.1, < 5"`

That would be better, IMO.

**Comment by Intrepidd** [comment]
There you go !

File: jquery-tablesorter.gemspec [hunk 1]
```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{vendor,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.markdown"]
 
-  s.add_dependency "railties", "~> 3.1"
+  s.add_dependency "railties", ">= 3.1, < 5
```
Comment (linjunpop) [inline jquery-tablesorter.gemspec]: Oops, missing `"`

Comment (Intrepidd) [inline jquery-tablesorter.gemspec]: My bad, fixing this right now

**Comment by Intrepidd** [comment]
There you go for real !

**Comment by linjunpop** [comment]
@Intrepidd Thanks. :beers:

**Pull request merged**